doozer/doozerlib/rhcos.py

```diff
@@ -184,15 +184,11 @@ def latest_container(self, container_conf: dict = None) -> Tuple[Optional[str],
         :param container_conf: a payload tag conf Model from group.yml (with build_metadata_key)
         :return: Returns (rhcos build id, image pullspec) or (None, None) if not found.
         """
-        if int(self.version.split('.')[0]) == 4 and int(self.version.split('.')[1]) >= 19:
-            stdout, _ = exectools.cmd_assert(f"oc image info {ART_PROD_IMAGE_REPO}:{self.version}-{self.runtime.group_config.vars['RHCOS_EL_MAJOR']}.{self.runtime.group_config.vars['RHCOS_EL_MINOR']}-node-image --filter-by-os {self.go_arch} -o json")
+        if self.runtime.group_config.rhcos.layered_rhcos:
+            stdout, _ = exectools.cmd_assert(f"oc image info {container_conf.rhcos_index_tag} --filter-by-os {self.go_arch} -o json")
```

Comment:
Why not using `doozerlib.util.oc_image_info()` instead? it is configurable with custom registry credentials.